deb bulk-extractor: Bump version to 1.5.5

mamedin opened the pull request:
* Bump version to 1.5.5
* Use http://downloads.digitalcorpora.org/downloads/bulk_extractor/
instead of github. Github repository has no release 1.5.5, so it is
needed to download the source tarball from download site for latest
releases.
* Check tarball sha256sum.

Connects to https://github.com/archivematica/Issues/issues/527

mamedin committed:
deb bulk-extractor: Bump version to 1.5.5

* Bump version to 1.5.5
* Use http://downloads.digitalcorpora.org/downloads/bulk_extractor/
instead of github. Github repository has no release 1.5.5, so it is
needed to download the source tarball from download site for latest
releases.
* Check tarball sha256sum

mamedin commented:
After merging this PR, CentOS, Xenial and Bionic will use the same `bulk-extractor` version (1.5.5, latest).

scollazo commented on debs/bionic/bulk_extractor/debian/changelog:
```diff
@@ -1,3 +1,20 @@
+bulk-extractor (1.5.5-1~18.04) xenial; urgency=low
```
```suggestion
bulk-extractor (1.5.5-1~18.04) bionic; urgency=low
```

scollazo reviewed (changes requested):
The Makefile for the bionic project is wrong,and uses "xenial".
```
cd /debbuild/$(NAME) && dch -v $(VERSION)-$(RELEASE) -D xenial "New upstream release"
```

Also, downloading a package without https, and that they don't provide hashes/signatures for the downloaded files, gives me goosegumps. I think we should hardcode the sha256sum for the tar.gz file in the script, and verify the downloaded file against it.

It's not a perfect solution, but...

mamedin commented:
Thanks @scollazo for the suggestions. I added a new var with the sha256sum to test the tarball and fixed the xenial/bionic typos.

Ready for CR again.

mamedin commented:
sha256sum for pubished packages:
```
ohbot@sites-pub:~/release-1.9.x-bionic-extras$ sha256sum bulk-extractor_1.5.5-1~18.04_amd64.deb 
3b1d4282dc6a51223957cb255b1d6fd5966e6b037175ae0e9c261f097c2bdbee  bulk-extractor_1.5.5-1~18.04_amd64.deb

ohbot@sites-pub:~/release-1.9.x-xenial-extras$ sha256sum bulk-extractor_1.5.5-1~16.04_amd64.deb 
f429345a83781a5e4580ff139e94ede627dd6728449973c98f2803d8ff0a2271  bulk-extractor_1.5.5-1~16.04_amd64.deb
```
